test/utils.spec.js

```diff
@@ -49,5 +49,13 @@ describe('utils', function () {
       expect(utils.getRandomCronForOnceEveryXDays(7)).to.eql('1 1 1,8,15,22 * *');
       expect(utils.getRandomCronForOnceEveryXDays(15)).to.eql('1 1 1,16 * *');
     });
+    it('should return a random cron expression for every x hours for the day', function () {
+      const AssertionError = require('assert').AssertionError;
+      expect(utils.getRandomCronForEveryDayAtXHoursInterval.bind(utils, 29)).to.throw(AssertionError);
+      //bind returns a ref to function which is executed and checked for if it threw exception.
+      expect(utils.getRandomCronForEveryDayAtXHoursInterval(8)).to.eql('1 1,9,17 * * *');
```

Comment:
just for my knowledge , how can we be sure that the getRandomInt function returns these specific values in UT ?

Reply:
its [mocked](https://github.com/jagadish-kb/service-fabrik-broker/blob/3b318621a0516420760bfc9c815b572ee8335028/test/utils.spec.js#L39) not to return random numbers :-)

Reply:
Ok , thanks :)